local-test/stress-multiproof.py

```
@@ -0,0 +1,309 @@
+    payload = _b64url(json.dumps({"iat": int(time.time())}).encode())
+    msg = f"{header}.{payload}".encode()
+    sig = hmac.new(secret, msg, hashlib.sha256).digest()
+    return f"{header}.{payload}.{_b64url(sig)}"
+
+
+def rpc(url: str, method: str, params: list, jwt: str | None = None) -> dict:
+    body = json.dumps({"jsonrpc": "2.0", "id": 1, "method": method, "params": params}).encode()
+    headers = {"Content-Type": "application/json"}
+    if jwt:
+        headers["Authorization"] = f"Bearer {jwt}"
+    req = urllib.request.Request(url, data=body, headers=headers)
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)